components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -233,6 +233,25 @@ const TransactionsOverviewSection = ({ host, isLoading, dateInterval }) => {
     },
   ];
 
+  /*
+   * If a date doesn't have any contributions or expenses API returns nothing.
+   * But we need to make sure the two series (expenses and contributions) show 0 in these cases rather than NaN which
+   * is shown by default by Appex charts.
+   */
+  if (series[0]?.data?.length > series[1].data?.length) {
```

Comment:
This is poor practice to coalesce to null and do a comparison like this `series[0]?.data?.length > null`, `null > eries[1].data?.length` or `null > null`. You should not try to do a comparison if `null`.

Reply:
I've checked for null beforehanded; should be handled now.